ansible/roles/ci/doc/how_does_it_work.md

```diff
@@ -0,0 +1,12 @@
+# How does it work
+
+The build tools was created to run on the [Docker](https://www.docker.com/) based CI servers.
+The Docker image for all build is one hosted on [Docker Hub](https://hub.docker.com/r/shadowrobot/ubuntu-ros-indigo-build-tools)
```

Comment:
"The Docker image for all builds are hosted on [Docker Hub](https://hub.docker.com/r/shadowrobot/ubuntu-ros-indigo-build-tools)" 

Is this already available? I can not see where are the docker images in the mentioned webpage.